submissions/servers/servers.md

```diff
@@ -0,0 +1,44 @@
+The reason for which Factorio uses it's version of the lockstep algorithm is that Factorio has a lot more inputs than Overwatch; there are logistic bots, miners, trains, spidertrons, biters, players and all of that adds up to a large amount of data that would need to be transferred every tick. Instead, it is much easier to just give the client the map when they connect and tell them any other inputs that would alter the simulation, such as a player moving 10 tiles to the right etc. See image below.
+
+{% include image.html src='./factorioserverjoin.png' %}
+
+A [Fully Deterministic algorithm](https://en.wikipedia.org/wiki/Deterministic_algorithm) is also used in Factorio. Such an algorithm will produce the same output when given the same input. This means that there is no randomness in the results, which is very useful in some cases such as Factorio. A Fully Deterministic algorithm is required when multiple instances of Factorio are run so all instances run in a lockstep algorithm and are in sync. A Fully Deterministic algorithm is defined by the following:
```

Comment:
This paragraph needs to start with why Determinism is required. That Determinism is required, _because_ of the LockStep architecture. There's no way to walk in lockstep if people walk in different ways.